packages/template-retail-react-app/app/pages/product-list/partials/color-refinements.jsx

```diff
@@ -50,10 +50,7 @@ const ColorRefinements = ({filter, toggleFilter, selectedFilters}) => {
                                     <Center
                                         {...styles.swatchButton}
                                         marginRight={0}
-                                        border={
-                                            value.label.toLowerCase() === 'white' &&
-                                            '1px solid black'
-                                        }
+                                        border={'1px solid black'}
```

Comment:
```suggestion
                                        border="1px solid black"
```
nit: if it's just not computed, you can just do a regular attribute.

Reply:
Fixed